packages/migration/src/utils/elasticsearch-helper.ts

```diff
@@ -27,7 +27,6 @@ export const updateComposition = async (
   try {
     response = await client.update({
       index: ELASTICSEARCH_INDEX_NAME,
-      type: 'compositions',
```

Comment:
@makelicious as our in-house Elasticsearch expert, do you see nothing wrong in removing the "type" fields in how we use Elasticsearch? As of my understanding this was to do with a schema mapping and us at one point defining a mapping type called `compositions`

Reply:
Perfect. I think at some point we need to introduce setting up the elastic index (with a full schema definition) whenever the reindexing happens. I feel like there are many cases in the schema ES has misinterpreted (searchable string vs keyword, ID fields for instance) field values and then on the code level we are fighting against these misinterpretations